group_vars/customer/eugene/development/secrets/registries.vault.yml

```diff
@@ -0,0 +1,7 @@
+# Docker private registries
+registries:
+  - name: foo
+    server: https://foo.com
+    username: foo
+    password: pass
+    email: foo@example.com
```

Comment:
may not be necessary?

Reply:
It's here for an example, I think we should keep it. And I think you will encrypt it before merging ?

Reply:
Yes, it's an example file that does not require to be encrypted.

Reply:
It's now encrypted and added to the CI so that docker registry credentials secrets are tested.